addons/binding/org.openhab.binding.networkcamera/README.md

```diff
@@ -0,0 +1,85 @@
+# Network Camera Binding
+
+This binding integrates large number of network cameras which can send images to a FTP server when motion or sound is detected.
+Binding acts as a FTP server.
+Images stored on the FTP server are not saved to the file system, therefore the binding shouldn't cause any problems on flash based openHAB installations.
+
+
+## Supported Things
+
+This binding supports ```motiondetection``` Thing. Every camera is identified by FTP user name. Therefore, every camera should use unique user name to login FTP server.
+
+## Discovery
+
+Automatic discovery is not supported.
+
+## Binding Configuration
+
+Bindings FTP server listening 2121 TCP port by default, but port can be configured. Also idle timeout can be configured.
+
+## Channels
+
+This binding currently supports following channels:
+
+| Channel Type ID | Item Type    | Description                                                                            |
+|-----------------|--------------|----------------------------------------------------------------------------------------|
+| image           | Image        | Image received from network camera.                                                    |
+
+
+### Trigger Channels
+
+| Channel Type ID | Options                | Description                                        |
+|-----------------|------------------------|----------------------------------------------------|
+| motion-trigger  | MOTION_DETECTED        | Triggered when image received from network camera. |
+
+
+## Full Example
+
+Things:
+
+```
+Thing networkcamera:motiondetection:garage [ userName="garage", password="12345" ]
+```
+
+Items:
+
+```
+Switch Garage_NetworkCamera_Motion       { channel="networkcamera:motiondetection:garage:motion" } 
+Image  Garage_NetworkCamera_Motion_Image { channel="networkcamera:motiondetection:garage:image" } 
+```
+
+Rules:
+
+```
+rule "example trigger rule"
```

Comment:
how about changing this example so it reacts on the state of the image item on the image channel, so the trigger channel can be removed?